.github/workflows/tests.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         run: |
           if [[ "${{ matrix.arch }}" == "rocm" ]]; then
             groups
-            /opt/rocm-5.4.3/bin/rocminfo
+            /opt/rocm/bin/rocminfo
```

Comment:
That does not appear to work :(

Reply:
I created a ticket for it https://it-support.mila.quebec/support/tickets/12948.
It seems the machine does not have rocm installed at all.
We can even see that we do not have the proper unix groups to use the GPUs.

Reply:
Ah, fair enough.